app/controllers/users_controller.rb

```diff
@@ -1,9 +1,10 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to put this helper in the application controller instead so that the user is authenticated across all controllers and pages, instead of only the user controller.

Reply:
@TedLivist Thank you so much, I will do it for sure.